packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/ResultOptionsBar/index.js

```diff
@@ -35,42 +35,48 @@ export class ResultOptionsBar extends Component {
     } = this.props
 
     return (
-      <Menu attached="top" style={{ position: 'relative' }} tabular>
-        {itemEnabled && (
+      <Menu
+        attached="top"
+        className="icon"
+        style={{ position: 'relative' }}
+        tabular
+      >
+        <Menu.Item
```

Comment:
add bool prop `link` to all Menu.Item (maybe it should be false if the item is active, you'll need to test that)